cibuildwheel/linux.py

```diff
@@ -24,6 +24,8 @@ def get_python_configurations(build_selector):
         PythonConfiguration(identifier='cp36-manylinux_i686', path='/opt/python/cp36-cp36m'),
         PythonConfiguration(identifier='cp37-manylinux_i686', path='/opt/python/cp37-cp37m'),
         PythonConfiguration(identifier='cp38-manylinux_i686', path='/opt/python/cp38-cp38'),
+        PythonConfiguration(identifier='pp27_73-manylinux_pypy_x86_64', path='/opt/python/pp273-pypy_73'),
+        PythonConfiguration(identifier='pp36_73-manylinux_pypy_x86_64', path='/opt/python/pp373-pypy36_pp73'),
```

Comment:
The `identifier` feels weird. I understand it eases the dispatch of the correct docker image later but this lacks consistency with other platforms as well. Since it is a user visible name, I'd rather it be consistent across platforms.
I would go with
```
PythonConfiguration(identifier='pp27_73-manylinux_x86_64', path='/opt/python/pp273-pypy_73'),
PythonConfiguration(identifier='pp36_73-manylinux_x86_64', path='/opt/python/pp373-pypy36_pp73'),
```
and introduce a bit more complexity when dispatching the docker image.

Reply:
YES! Great catch. I'd actually forgotten I'd done this, but this is way better!